spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/procedures/CreateChangelogViewProcedure.java

```diff
@@ -179,13 +187,24 @@ private boolean shouldRemoveCarryoverRows(ProcedureInput input) {
     return input.asBoolean(REMOVE_CARRYOVERS_PARAM, true);
   }
 
-  private Dataset<Row> removeCarryoverRows(Dataset<Row> df) {
+  private Dataset<Row> removeCarryoverRows(Dataset<Row> df, boolean netChanges) {
+    Predicate<String> columnsToRemove;
+    if (netChanges) {
+      columnsToRemove =
+          column ->
```

Comment:
I'd probably change this to a set contains check

Reply:
NetChangeRemovals::contains

Reply:
Can you be more specific?

Reply:
I think @RussellSpitzer means define a set outside of CHANGE_TYPE, CHANGE_ORDINAL, COMMIT_SNAPSHOT_ID?

Reply:
Created a set per discussion, put the set within the method since it is not used by somewhere else.